packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/summary.css

```diff
@@ -15,6 +15,16 @@
     grid-template-columns: 1fr;
 }
 
+.edit_button {
+    display: grid;
```

Comment:
Unfortunately, `button` elements don't work consistently as `grid` or even `flex` containers, so this needs a child `span` to serve as the grid container.

I recommend using our `LinkButton` component for these.

Reply:
Rebuild to use `LinkButton` component, thank you